### ISSUE TITLE
React component fires events for `undefined` events

### ISSUE THREAD

**Issue by lexanth** [nhn/tui.editor#2201]
## Describe the bug
When using the react component, if an explicit `undefined` is provided for an event handler, it still registers the event handler which then fails when the event is triggered.

## To Reproduce
Create an `Editor` using the react wrapper and provide an explicit `undefined` as an event hander (e.g. `onBlur`)
Trigger the event.

e.g.
```tsx
<Editor onBlur={props.onBlur} ... />
```
If my component (which wraps the TUI editor) has an optional `props.onBlur` handler, `Editor` always receives an `onBlur` prop, but it may be `undefined`.

The workaround is to do something like
```tsx
<Editor {...(props.onBlur ? { onBlur : props.onBlur } : {})} ... />
```
(which is fine as long as whether `onBlur` is provided doesn't change during the component's lifecycle)

## Expected behavior
TUI should ignore `undefined` events when registering event handlers

## Desktop (please complete the following information):
 - OS: macOS
 - Browser: Chrome
 - Version 97

## Additional context
Missing and explicit `undefined` are slightly different things. TS has newly added https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes which differentiates between them, but hopefully this demonstrates that it would be a useful change

The relevant code is https://github.com/nhn/tui.editor/blob/master/apps/react-editor/src/editor.tsx#L18-L39

These pieces would just have to filter out falsy events.

**Comment by js87zz** [comment]
@lexanth 
I checked the issue. 
I think you can resolve if you don't pass `undefined` events in your component.
I'll let you know when it's fixed.

**Comment by js87zz** [comment]
@lexanth 
The bug has been fixed in `v3.1.3`(#2292)
